packages/types-algosdk/index.d.ts

```diff
@@ -221,6 +257,20 @@ declare module 'algosdk' {
    */
   function tealSignFromProgram(sk: Uint8Array, data: Uint8Array, program: Uint8Array): Uint8Array;
 
+  export function makeApplicationClearStateTxn(from: any, suggestedParams: any, appIndex: any, appArgs?: any, accounts?: any, foreignApps?: any, foreignAssets?: any, note?: any, lease?: any, rekeyTo?: any): any;
```

Comment:
if possible, please try to not use `any`. At least `from`, `appIndex`, `accounts`, `note`, `lease` ... can have  a proper type. Look at the commit I merged recenlty  - I added few useful types like: `LogicSigArgs` and `Address`.

Reply:
ok